<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="toDoTab">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="encodeTab">
       <attribute name="title">
        <string>Encode methods</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="mainGridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <item row="0" column="1">
           <layout class="QGridLayout" name="hostWatermarkLayout">
            <item row="0" column="0" rowspan="2">
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="watermarkSelectionButton">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="toolTip">
               <string>Load host audio file </string>
              </property>
              <property name="text">
               <string>Select Watermark base file ...</string>
              </property>
              <property name="icon">
               <iconset resource="icons/qstyle.qrc">
                <normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-32.png</normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTextEdit" name="informationHostWatermark">
              <property name="palette">
               <palette>
                <active>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="Base">
                  <brush brushstyle="SolidPattern">
                   <color alpha="255">
                    <red>240</red>
                    <green>240</green>
                    <blue>240</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; color:#ffffff;&quot;&gt;Here will be printed information about the loaded host file and/or some logs about computational part of the program.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QGridLayout" name="paramsLayout">
            <item row="0" column="2">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;lt;- Method selection&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="QTabWidget" name="selectingMethodTab">
              <property name="toolTip">
               <string/>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="lsbMethodTab">
               <property name="toolTip">
                <string/>
               </property>
               <attribute name="title">
                <string>LSB Method</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="1" column="2">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows you to save current configuration as a script &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows you to load a configuration script &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="2">
                   <widget class="QPushButton" name="lsbLoadConfigurationButton">
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="toolTip">
                     <string>load a configuration script</string>
                    </property>
                    <property name="statusTip">
                     <string/>
                    </property>
                    <property name="whatsThis">
                     <string/>
                    </property>
                    <property name="text">
                     <string>Load configuration script ...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="icons/qstyle.qrc">
                      <normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-16.png</normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-16.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QPushButton" name="lsbSaveConfigurationButton">
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="toolTip">
                     <string>save current onfiguration as a script</string>
                    </property>
                    <property name="text">
                     <string>Save configuration script ...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="icons/qstyle.qrc">
                      <normaloff>:/trolltech/styles/commonstyle/images/floppy-16.png</normaloff>:/trolltech/styles/commonstyle/images/floppy-16.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="3">
                   <widget class="QLabel" name="label_2">
                    <property name="autoFillBackground">
                     <bool>false</bool>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Here is the configuration tab for the LSB method. May be useful to define editable parameters such as:&lt;/p&gt;&lt;p&gt;- sample size&lt;/p&gt;&lt;p&gt;- number of less significant bits used for watermarking datas (1 by default)&lt;/p&gt;&lt;p&gt;- any idea ?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="sswMethodTab">
               <property name="toolTip">
                <string/>
               </property>
               <attribute name="title">
                <string>SSW Method</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="1" column="2">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows you to save current configuration as a script &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows you to load a configuration script &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="2">
                   <widget class="QPushButton" name="sswLoadConfigurationButton">
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="toolTip">
                     <string>load a configuration script</string>
                    </property>
                    <property name="text">
                     <string>Load configuration script ...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="icons/qstyle.qrc">
                      <normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-16.png</normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-16.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QPushButton" name="sswSaveConfigurationButton">
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="toolTip">
                     <string>save current onfiguration as a script</string>
                    </property>
                    <property name="text">
                     <string>Save configuration script ...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="icons/qstyle.qrc">
                      <normaloff>:/trolltech/styles/commonstyle/images/floppy-16.png</normaloff>:/trolltech/styles/commonstyle/images/floppy-16.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="3">
                   <widget class="QLabel" name="label_6">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Here is the configuration tab for the SSW method. May be useful to define editable parameters such as:&lt;/p&gt;&lt;p&gt;- any idea ?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="compExpMethodTab">
               <property name="toolTip">
                <string/>
               </property>
               <attribute name="title">
                <string>Compression-Expansion Method</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_9">
                  <item row="1" column="2">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows you to save current configuration as a script &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Allows you to load a configuration script &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0" colspan="2">
                   <widget class="QPushButton" name="compExpLoadConfigurationButton">
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="toolTip">
                     <string>load a configuration script</string>
                    </property>
                    <property name="text">
                     <string>Load configuration script ...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="icons/qstyle.qrc">
                      <normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-16.png</normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-16.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QPushButton" name="compExpSaveConfigurationButton">
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="toolTip">
                     <string>save current onfiguration as a script</string>
                    </property>
                    <property name="text">
                     <string>Save configuration script ...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="icons/qstyle.qrc">
                      <normaloff>:/trolltech/styles/commonstyle/images/floppy-16.png</normaloff>:/trolltech/styles/commonstyle/images/floppy-16.png</iconset>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="3">
                   <widget class="QLabel" name="label_9">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="text">
                     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Here is the configuration tab for the Compression-Expansion method. May be useful to define editable parameters such as:&lt;/p&gt;&lt;p&gt;- any idea ?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="3" column="0" colspan="3">
             <widget class="QPushButton" name="encodeButton">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Encode ...</string>
              </property>
              <property name="icon">
               <iconset resource="icons/qstyle.qrc">
                <normaloff>:/trolltech/styles/commonstyle/images/desktop-32.png</normaloff>:/trolltech/styles/commonstyle/images/desktop-32.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QComboBox" name="selectingMethodComboBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="iconSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>LSB Method</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SSW Method</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Compression-Expansion Method</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="watermarkLayout">
            <item row="2" column="0">
             <widget class="QProgressBar" name="usedWatermarkCapacity">
              <property name="value">
               <number>85</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTextEdit" name="textToWatermark">
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Here is the editable text that will be watermarked in the host file. The progress bar shows the available amount of data that you can put on the loaded host file.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="decodeTab">
       <attribute name="title">
        <string>Decode Methods</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QTabWidget" name="selectingMethodTab_2">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_7">
             <attribute name="title">
              <string>LSB Method</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_8">
             <attribute name="title">
              <string>SSW Method</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_9">
             <attribute name="title">
              <string>Compression-Expansion Method</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="watermarkSelectionButton_2">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Select Watermark base file ...</string>
            </property>
            <property name="icon">
             <iconset resource="icons/qstyle.qrc">
              <normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-32.png</normaloff>:/trolltech/styles/commonstyle/images/standardbutton-open-32.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="decodeButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Decode ...</string>
            </property>
            <property name="icon">
             <iconset resource="icons/qstyle.qrc">
              <normaloff>:/trolltech/styles/commonstyle/images/filecontents-32.png</normaloff>:/trolltech/styles/commonstyle/images/filecontents-32.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="degradationTab">
       <attribute name="title">
        <string>Degradation methods</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_watermark_base_file"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHow_to"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHow_to">
   <property name="text">
    <string>How to</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit (CTRL+Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad_watermark_base_file">
   <property name="text">
    <string>Load watermark base file ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/qstyle.qrc"/>
 </resources>
 <connections/>
</ui>
